poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
```

Comment:
I don't know why poetry and dependencies not directly related to `accelerate` had their version updated when I ran `poetry lock`. Would you happen to know more about this @aliberts ? How much of an impact it has ?

Reply:
Yes, you need to run `poetry lock --no-update` because `lock` will update dependencies by default

Reply:
ok perfect thanks ! 🤗